docs/usage/loading-packages.md

```diff
@@ -3,18 +3,17 @@
 
 Only the Python standard library is available after importing Pyodide.
 To use other packages, you’ll need to load them using either:
- - {ref}`pyodide.loadPackage <js_api_pyodide_loadPackage>` for packages built
-   with pyodide, or 
+ - {any}`pyodide.loadPackage` for packages built with pyodide, or 
  - `micropip.install` for pure Python packages with wheels available on PyPi or
    from other URLs.
 
 ```{note}
 `micropip` can also be used to load packages built in pyodide (in
-which case it relies on {ref}`pyodide.loadPackage <js_api_pyodide_loadPackage>`).
+which case it relies on {any}`pyodide.loadPackage`).
```

Comment:
Wouldn't we have a name collision if a Python function has the same name?

Reply:
It might be a good idea to call these things `pyodide_js` and `pyodide_py`. Also, we should not name Python functions the same things as the Javascript functions, this will be confusing. Any name with multiple words should be fine since `snake_case` and `snakeCase` don't clash and it's obvious which API is which.